ansible_base/oauth2_provider/serializers/token.py

```diff
@@ -0,0 +1,101 @@
+import logging
+from datetime import timedelta
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.utils.timezone import now
+from django.utils.translation import gettext_lazy as _
+from oauthlib.common import generate_token
+from oauthlib.oauth2 import AccessDeniedError
+from rest_framework.exceptions import PermissionDenied, ValidationError
+from rest_framework.serializers import SerializerMethodField
+
+from ansible_base.lib.serializers.common import CommonModelSerializer
+from ansible_base.lib.utils.encryption import ENCRYPTED_STRING
+from ansible_base.lib.utils.settings import get_setting
+from ansible_base.oauth2_provider.models import OAuth2AccessToken, OAuth2RefreshToken
+
+logger = logging.getLogger("ansible_base.serializers.oauth2_provider")
+
+
+class BaseOAuth2TokenSerializer(CommonModelSerializer):
+    refresh_token = SerializerMethodField()
+    token = SerializerMethodField()
+    ALLOWED_SCOPES = ['read', 'write']
```

Comment:
Should we pull this out of the scope field from the model so that if we ever changed that this would auto-magically update?